Sources/PubNubChatComponents/Views/Reaction/MessageReactionListView.swift

```diff
@@ -28,61 +28,40 @@
 import UIKit
 import Combine
 
-public class MessageReactionListComponent: UIView {//UIStackContainerView {
-
-  // 👍 thumbs up U+1F44D
-  lazy public var thumbsUpReactionView = MessageReactionButtonComponent(type: .custom)
-  // ❤️ red heart U+2764
-  lazy public var redHeartReactionView = MessageReactionButtonComponent(type: .custom)
-  // 😂 face with tears of joy U+1F602
-  lazy public var faceWithTearsOfJoyReactionView = MessageReactionButtonComponent(type: .custom)
-  // 😲 astonished face U+1F632
-  lazy public var astonishedFaceReactionView = MessageReactionButtonComponent(type: .custom)
-  // 😢 crying face U+1F622
-  lazy public var cryingFaceReactionView = MessageReactionButtonComponent(type: .custom)
-  // 🔥 fire U+1F525
-  lazy public var fireReactionView = MessageReactionButtonComponent(type: .custom)
-
-  var currentCount: Int = 0
+public class MessageReactionListComponent: UIView {
+  private let provider: ReactionProvider
+  private var reactionButtons: [MessageReactionButtonComponent] = []
```

Comment:
It it possible to make `reactionButtons` as a `let` variable?

Reply:
I was thinking about making it dynamic, so if we have 2 emoji in message then only 2 buttons are in the list, if we do not have any than list is empty.